keyboards/scottokeebs/scottowing/keymaps/default/config.h

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2024 Joe Scotto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+// Define options
+#define TAPPING_TERM 135
+#define PERMISSIVE_HOLD
+#define TAPPING_TERM_PER_KEY
+
+// Bootmagic
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 1
```

Comment:
This is not something that should be applied at the keymap level. If it is needed/intended, it should be placed at the keyboard level `info.json`.
```suggestion
```

Reply:
This is needed for this board, added it to the `info.json` in [63d994](https://github.com/qmk/qmk_firmware/pull/23513/commits/263d99449e875e46419ced1bbb1d758a2999aa3e)